datasets/pcfg/data_generation.py

```diff
@@ -118,10 +117,10 @@ def generate_intermediate_tasks(source_line, target_line):
   last_task = intermediate_tasks.pop()
   intermediate_tasks.append(last_task + ' END')
 
-  return intermediate_tasks
+  return intermediate_tasks, len(operations)
```

Comment:
NIt: We should probably update the "Returns:" section of the docstring to mention `len(operations)`.